elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala

```diff
@@ -42,6 +42,10 @@ class RestlasticSearchClientTest extends WordSpec with Matchers with ScalaFuture
     new RestlasticSearchClient(new StaticEndpoint(new Endpoint(host, port)))
   }
 
+  override def refresh(): Unit = {
+    Await.result(restClient.refresh(index), 2.seconds)
```

Comment:
https://github.com/SumoLogic/elasticsearch-client/pull/33/commits/1078cfa469aa089441e0fc4b0fbfe89038cc7ad1  Does that do what you had in mind?  The method gets called less now as a side-effect, and renaming the original `refresh()` method to `refreshWithClient()` doesn't feel appropriate